.gitignore

```diff
@@ -11,3 +11,4 @@ logs/
 *.tar
 .gdb_history
 .DS_Store
+serial.log
```

Comment:
I don't think we need that, do we?

The local logs dir should be ignored.

Reply:
Fixed